changelog.md

```diff
@@ -1,2 +1,11 @@
-#v0.1
+v0.1.1
+------
+
+* Fix a deployment bug by adding `MANIFEST.in`.
+* Add a missing requirement for `django`.
```

Comment:
> why?

Several reasons, generally because we test against several versions of django. That means that:
- on travis, it will be installed only to be instantly replaced.
- it's something external that will be updated often, and will require fixing

but mostly:

- there is no one required version that is "required", so adding it to the requirements doesn't really make sense.

Hm. Perhaps we should be using tox.